packages/shared/src/components/filters/FeedFilters.tsx

```diff
@@ -1,21 +1,17 @@
 import React, { ReactElement, useState } from 'react';
-import classNames from 'classnames';
-import { ModalProps, StyledModal } from '../modals/StyledModal';
-import SidebarList from '../sidebar/SidebarList';
```

Comment:
@sshanzel Now that you also removed SteppedModal, `<StyledModal />` can also be removed right?

Reply:
Done! Finally, this file is gone. Woohoo! 🚀